.travis.yml

```diff
@@ -76,3 +77,10 @@ script:
   - BISON=${BISON} FLEX=${FLEX} make -j4
   - BISON=${BISON} FLEX=${FLEX} make test
   - make test_example
+
+  # build flex/bison files in CI with debug mode
+  - BISON=${BISON} FLEX=${FLEX} make cleanall
+  # relaxed builds for debug mode with current gcc versions, see #218
+  - BISON=${BISON} FLEX=${FLEX} make -j4 mode=debug relaxed_build=${RELAXED_MODE}
```

Comment:
To reduce confusion, how about harmonizing `RELAXED_MODE` and `relaxed_build`, e.g., by changing the env name to `RELAXED_BUILD`?